local_db.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
this is exactly the same as in the ARD repo, other than it using a different database. The nice thing about this is that there will only be one MySQL Docker container hanging around, so it should be easy to check things out from the same mysql session